### PR TITLE
feat(filters): add industries dropdown

### LINKED ISSUE #430: Topic page: add ability to filter by industry
Add an additional dropdown next to "Products" which allows the user to further filter down by industries as defined in the taxonomy.

@avanishgandhi

### LINKED ISSUE #433: Filters: design touchup
- [x] Wrong font on dropdown button
 image
- [x] Contrast and color of active dropdown button
image
- [x] Wrong font and color on search box
image

### PR DESCRIPTION
**Pull request by rofe**
Fix #430
Fix #433 

Test URL: https://multi-dropdown-filter--theblog--adobe.hlx.page/en/topics/news.html#

### CODE REVIEW

**Comment by rofe** [comment]
The first step was to refactor the filtering code to be capable of handling multiple dropdowns. For now the output should be the same as before, except for the dropdown title, which is now the category from the taxonomy doc.

**Comment by rofe** [comment]
The industries dropdown is now in; however, it looks like we're currently `OR`ing everything, while we should use `OR` within each dropdown, but `AND` between the two dropdowns. We need to adjust the query accordingly.

**Comment by rofe** [comment]
Actually, we should to use:
- `OR` within products
- `AND` between products and all topics, including industries and the main topic

**Comment by rofe** [comment]
- Query algorithm is now: `maintopic AND (product1 OR product2 OR ...) AND (industry1 OR industry2 OR ...)`
- Empty result page now implemented as designed
- Showing list of selected filters underneath the dropdowns (design input needed)

File: scripts/taxonomy.js [hunk 1]
```diff
@@ -119,6 +119,10 @@ export async function getTaxonomy() {
 
         getCategory: function (cat) {
           return this.node.querySelector(`[data-type="${cat}"]`);
+        },
+
+        getCategoryTitle: function (cat) {
+          return this.node.querySelector(`[data-type="${cat}"] > p`).innerText;
```
Comment (kptdobe) [inline scripts/taxonomy.js:125]: Missing a `trim` to be sure.

Comment (kptdobe) [reply scripts/taxonomy.js:125]: Where is the method used ?

**Review by kptdobe — APPROVED** [review]

File: scripts/taxonomy.js (hunk 1 above)
Comment (rofe) [reply scripts/taxonomy.js:125]: Here: https://github.com/adobe/theblog/blob/98d27588ecdd9e610c2bda1929e357957505ee87/scripts/filters.js#L210 (missed a file? 😉)

Comment (kptdobe) [reply scripts/taxonomy.js]: Weird, too many changes in filters.js, it was collapsed!

Comment (rofe) [reply scripts/taxonomy.js]: Yeah, I noticed that, too. Not helfpul 😄

**Pull request merged**